### DISCUSSION TITLE
Avatar and Avatar Group RFC

### DISCUSSION THREAD

**Discussion by nkrantz** [twilio-labs/paste#3195] [Review request]
_Attention engineers: if you haven't yet had a chance to review the proposed Avatar Group API, ➡️  [take a look at that RFC here](https://github.com/twilio-labs/paste/discussions/3153) ⬅️ and feel free to leave your feedback!_

# Avatar Group

Paste is adding an Avatar Group component to our [Avatar](https://paste.twilio.design/components/avatar) package and we'd like your input. See [the proposed spec doc here](https://docs.google.com/document/d/1EV12u4RgVggSmf4bO8B7OIjkLZZEfm-XEDyZvUAE2QM/edit#) (leave comments/suggestions/questions directly in the google doc) and see below for more information about what type of design feedback we're looking for.

[See the design exploration Figma here](https://github.com/twilio-labs/paste/discussions/3195). Feel free to leave design feedback in this thread, or on the Figma file.
![Overview of Avatar Group design exploration](https://user-images.githubusercontent.com/75342690/234344849-7eb9c4df-fb04-40c0-995d-2d8cb0fd8137.png)
As described in the above screenshot, we're looking for feedback on two design questions in particular:
1. What's the best way to compose an overflow Avatar Group state using existing Paste components (e.g. the "+5" indicator)?
![Avatar Group design exploration - overflow state](https://user-images.githubusercontent.com/75342690/234347500-7521e05c-0161-4c5e-9126-cdd12cf3ea14.png)
2. What's the ideal spacing to use between Avatars that are in a group?
![Avatar Group design exploration - spacing](https://user-images.githubusercontent.com/75342690/234347262-fdbc0b56-08c1-4eee-a48b-dfb04bfab1b5.png)

Thanks in advance for your input!

**Comment by TheSisb** [comment]
The component has been released and the window for comment has been closed.
